### ISSUE TITLE
setlocale: LC_ALL: cannot change locale

### ISSUE THREAD

**Issue by omega3** [anbox/anbox#247]
```
omega3omega3-945P-DS3:~$ snap install --classic anbox-installer && anbox-installer
error: access denied (try with sudo)
omega3omega3-945P-DS3:~$ sudo snap install --classic anbox-installer && anbox-installer
[sudo] hasło użytkownika omega3: 
Niestety, proszę spróbować ponownie.
[sudo] hasło użytkownika omega3: 
snap "anbox-installer" is already installed, see "snap refresh --help"
Anbox (Android in a Box) - Installer


IMPORTANT: THIS IS ALPHA LEVEL SOFTWARE. EXPECT INSTABILITY AND
           BUGS !!!!!

IMPORTANT: ALSO PLEASE BE AWARE THAT WE DON'T PROVIDE FULL
           CONFINEMENT FOR THE SNAP YET !!!!


PLEASE NOTE: This script will require root access on your system
to install all necessary things. It will prompt you to enter your
password when required.



What do you want to do?

 1. Install Anbox
 2. Uninstall Anbox

Please enter your choice [1-2]: 
1

This is the installer for the anbox runtime environment. It will
install certain things on your system to ensure all requirements
are available for anbox to work correctly.

In summary we will install the following things:

 * Add the anbox-support ppa ppa:morphis/anbox-support to the
   host system
 * Install the anbox-modules-dkms deb package from the ppa
   which will add kernel modules for ashmem and binder which are
   required for the Android container to work.
 * Configure binder and ashmem kernel modules to be loaded
   automatically on boot.
 * Install the anbox-common package from the ppa which will
   - Add an upstart job for the current user omega3 which will
     start the anbox runtime on login.
   - Add a X11 session configuration file to allow the system
     application launcher (Unity7, Gnome Shell, ..) to find
     available Android applications.

Please type 'I AGREE' followed by pressing ENTER to continue
or type anything else to abort:
I AGREE


Starting installation process ...

+ sudo apt install -y software-properties-common linux-headers-generic
Czytanie list pakietów... Gotowe
Budowanie drzewa zależności       
Odczyt informacji o stanie... Gotowe
software-properties-common is already the newest version (0.96.20.6).
Następujące pakiety zostały zainstalowane automatycznie i nie są już więcej wymagane:
  linux-headers-4.4.0-71 linux-headers-4.4.0-71-generic linux-image-4.4.0-71-generic
  linux-image-extra-4.4.0-71-generic linux-tools-4.4.0-71 linux-tools-4.4.0-71-generic snap-confine
Aby je usunąć należy użyć "sudo apt autoremove".
The following additional packages will be installed:
  linux-generic linux-headers-4.4.0-77 linux-headers-4.4.0-77-generic linux-image-4.4.0-77-generic
  linux-image-extra-4.4.0-77-generic linux-image-generic
Sugerowane pakiety:
  fdutils linux-doc-4.4.0 | linux-source-4.4.0
Zostaną zainstalowane następujące NOWE pakiety:
  linux-headers-4.4.0-77 linux-headers-4.4.0-77-generic linux-image-4.4.0-77-generic
  linux-image-extra-4.4.0-77-generic
Następujące pakiety zostaną zaktualizowane:
  linux-generic linux-headers-generic linux-image-generic
3 aktualizowanych, 4 nowo instalowanych, 0 usuwanych i 389 nieaktualizowanych.
Konieczne pobranie 68,6 MB archiwów.
Po tej operacji zostanie dodatkowo użyte 297 MB miejsca na dysku.
Pobieranie:1 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 linux-image-4.4.0-77-generic amd64 4.4.0-77.98 [21,9 MB]
Pobieranie:2 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 linux-image-extra-4.4.0-77-generic amd64 4.4.0-77.98 [35,9 MB]
Pobieranie:3 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 linux-generic amd64 4.4.0.77.83 [1786 B] 
Pobieranie:4 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 linux-image-generic amd64 4.4.0.77.83 [2306 B]
Pobieranie:5 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 linux-headers-4.4.0-77 all 4.4.0-77.98 [9979 kB]
Pobieranie:6 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 linux-headers-4.4.0-77-generic amd64 4.4.0-77.98 [784 kB]
Pobieranie:7 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 linux-headers-generic amd64 4.4.0.77.83 [2278 B]
Pobrano 68,6 MB w 57s (1190 kB/s)                                                                                
Wybieranie wcześniej niewybranego pakietu linux-image-4.4.0-77-generic.
(Odczytywanie bazy danych ... 632098 plików i katalogów obecnie zainstalowanych.)
Przygotowywanie do rozpakowania pakietu .../linux-image-4.4.0-77-generic_4.4.0-77.98_amd64.deb ...
Done.
Rozpakowywanie pakietu linux-image-4.4.0-77-generic (4.4.0-77.98) ...
Wybieranie wcześniej niewybranego pakietu linux-image-extra-4.4.0-77-generic.
Przygotowywanie do rozpakowania pakietu .../linux-image-extra-4.4.0-77-generic_4.4.0-77.98_amd64.deb ...
Rozpakowywanie pakietu linux-image-extra-4.4.0-77-generic (4.4.0-77.98) ...
Przygotowywanie do rozpakowania pakietu .../linux-generic_4.4.0.77.83_amd64.deb ...
Rozpakowywanie pakietu linux-generic (4.4.0.77.83) nad (4.4.0.75.81) ...
Przygotowywanie do rozpakowania pakietu .../linux-image-generic_4.4.0.77.83_amd64.deb ...
Rozpakowywanie pakietu linux-image-generic (4.4.0.77.83) nad (4.4.0.75.81) ...
Wybieranie wcześniej niewybranego pakietu linux-headers-4.4.0-77.
Przygotowywanie do rozpakowania pakietu .../linux-headers-4.4.0-77_4.4.0-77.98_all.deb ...
Rozpakowywanie pakietu linux-headers-4.4.0-77 (4.4.0-77.98) ...
Wybieranie wcześniej niewybranego pakietu linux-headers-4.4.0-77-generic.
Przygotowywanie do rozpakowania pakietu .../linux-headers-4.4.0-77-generic_4.4.0-77.98_amd64.deb ...
Rozpakowywanie pakietu linux-headers-4.4.0-77-generic (4.4.0-77.98) ...
Przygotowywanie do rozpakowania pakietu .../linux-headers-generic_4.4.0.77.83_amd64.deb ...
Rozpakowywanie pakietu linux-headers-generic (4.4.0.77.83) nad (4.4.0.75.81) ...
Konfigurowanie pakietu linux-image-4.4.0-77-generic (4.4.0-77.98) ...
Running depmod.
update-initramfs: deferring update (hook will be called later)
Examining /etc/kernel/postinst.d.
run-parts: executing /etc/kernel/postinst.d/apt-auto-removal 4.4.0-77-generic /boot/vmlinuz-4.4.0-77-generic
run-parts: executing /etc/kernel/postinst.d/dkms 4.4.0-77-generic /boot/vmlinuz-4.4.0-77-generic
run-parts: executing /etc/kernel/postinst.d/initramfs-tools 4.4.0-77-generic /boot/vmlinuz-4.4.0-77-generic
update-initramfs: Generating /boot/initrd.img-4.4.0-77-generic
run-parts: executing /etc/kernel/postinst.d/pm-utils 4.4.0-77-generic /boot/vmlinuz-4.4.0-77-generic
run-parts: executing /etc/kernel/postinst.d/unattended-upgrades 4.4.0-77-generic /boot/vmlinuz-4.4.0-77-generic
run-parts: executing /etc/kernel/postinst.d/update-notifier 4.4.0-77-generic /boot/vmlinuz-4.4.0-77-generic
run-parts: executing /etc/kernel/postinst.d/zz-update-grub 4.4.0-77-generic /boot/vmlinuz-4.4.0-77-generic
Tworzenie pliku konfiguracyjnego grub...
Znaleziono motyw: /boot/grub/themes/breeze/theme.txt
Uwaga: Setting GRUB_TIMEOUT to a non-zero value when GRUB_HIDDEN_TIMEOUT is set is no longer supported.
Found background image: /usr/share/images/desktop-base/desktop-grub.png
Znaleziono obraz Linuksa: /boot/vmlinuz-4.4.0-77-generic
Znaleziono obraz initrd: /boot/initrd.img-4.4.0-77-generic
Znaleziono obraz Linuksa: /boot/vmlinuz-4.4.0-75-generic
Znaleziono obraz initrd: /boot/initrd.img-4.4.0-75-generic
Znaleziono obraz Linuksa: /boot/vmlinuz-4.4.0-72-generic
Znaleziono obraz initrd: /boot/initrd.img-4.4.0-72-generic
Znaleziono obraz Linuksa: /boot/vmlinuz-4.4.0-71-generic
Znaleziono obraz initrd: /boot/initrd.img-4.4.0-71-generic
Found memtest86+ image: /boot/memtest86+.elf
Found memtest86+ image: /boot/memtest86+.bin
gotowe
Konfigurowanie pakietu linux-image-extra-4.4.0-77-generic (4.4.0-77.98) ...
run-parts: executing /etc/kernel/postinst.d/apt-auto-removal 4.4.0-77-generic /boot/vmlinuz-4.4.0-77-generic
run-parts: executing /etc/kernel/postinst.d/dkms 4.4.0-77-generic /boot/vmlinuz-4.4.0-77-generic
run-parts: executing /etc/kernel/postinst.d/initramfs-tools 4.4.0-77-generic /boot/vmlinuz-4.4.0-77-generic
update-initramfs: Generating /boot/initrd.img-4.4.0-77-generic
run-parts: executing /etc/kernel/postinst.d/pm-utils 4.4.0-77-generic /boot/vmlinuz-4.4.0-77-generic
run-parts: executing /etc/kernel/postinst.d/unattended-upgrades 4.4.0-77-generic /boot/vmlinuz-4.4.0-77-generic
run-parts: executing /etc/kernel/postinst.d/update-notifier 4.4.0-77-generic /boot/vmlinuz-4.4.0-77-generic
run-parts: executing /etc/kernel/postinst.d/zz-update-grub 4.4.0-77-generic /boot/vmlinuz-4.4.0-77-generic
Tworzenie pliku konfiguracyjnego grub...
Znaleziono motyw: /boot/grub/themes/breeze/theme.txt
Uwaga: Setting GRUB_TIMEOUT to a non-zero value when GRUB_HIDDEN_TIMEOUT is set is no longer supported.
Found background image: /usr/share/images/desktop-base/desktop-grub.png
Znaleziono obraz Linuksa: /boot/vmlinuz-4.4.0-77-generic
Znaleziono obraz initrd: /boot/initrd.img-4.4.0-77-generic
Znaleziono obraz Linuksa: /boot/vmlinuz-4.4.0-75-generic
Znaleziono obraz initrd: /boot/initrd.img-4.4.0-75-generic
Znaleziono obraz Linuksa: /boot/vmlinuz-4.4.0-72-generic
Znaleziono obraz initrd: /boot/initrd.img-4.4.0-72-generic
Znaleziono obraz Linuksa: /boot/vmlinuz-4.4.0-71-generic
Znaleziono obraz initrd: /boot/initrd.img-4.4.0-71-generic
Found memtest86+ image: /boot/memtest86+.elf
Found memtest86+ image: /boot/memtest86+.bin
gotowe
Konfigurowanie pakietu linux-image-generic (4.4.0.77.83) ...
Konfigurowanie pakietu linux-headers-4.4.0-77 (4.4.0-77.98) ...
Konfigurowanie pakietu linux-headers-4.4.0-77-generic (4.4.0-77.98) ...
Examining /etc/kernel/header_postinst.d.
run-parts: executing /etc/kernel/header_postinst.d/dkms 4.4.0-77-generic /boot/vmlinuz-4.4.0-77-generic
Konfigurowanie pakietu linux-headers-generic (4.4.0.77.83) ...
Konfigurowanie pakietu linux-generic (4.4.0.77.83) ...
+ sudo add-apt-repository -y ppa:morphis/anbox-support
gpg: zbiór kluczy ,,/tmp/tmpydejr325/secring.gpg'' został utworzony
gpg: zbiór kluczy ,,/tmp/tmpydejr325/pubring.gpg'' został utworzony
gpg: zapytanie o klucz 875B67B7 w serwerze hkp keyserver.ubuntu.com
gpg: /tmp/tmpydejr325/trustdb.gpg: baza zaufania utworzona
gpg: klucz 875B67B7: zaimportowano klucz publiczny ,,Launchpad PPA for morphis''
gpg: Ogółem przetworzonych kluczy: 1
gpg:               dołączono do zbioru: 1  (RSA: 1)
OK
+ sudo apt update
Pobieranie:1 http://security.ubuntu.com/ubuntu xenial-security InRelease [102 kB]
Stary:2 http://archive.ubuntu.com/ubuntu xenial InRelease                                                        
Pobieranie:3 http://archive.ubuntu.com/ubuntu xenial-updates InRelease [102 kB]                                  
Stary:4 http://ppa.launchpad.net/embrosyn/cinnamon/ubuntu xenial InRelease                                       
Stary:5 http://ppa.launchpad.net/graphics-drivers/ppa/ubuntu xenial InRelease                                    
Ign.:6 http://repo.vivaldi.com/stable/deb stable InRelease                                                       
Stary:7 http://ppa.launchpad.net/lubuntu-dev/lubuntu-daily/ubuntu xenial InRelease                               
Stary:8 http://repo.vivaldi.com/stable/deb stable Release                                                        
Stary:9 http://ppa.launchpad.net/me-davidsansome/clementine/ubuntu xenial InRelease                              
Stary:10 http://ppa.launchpad.net/morphis/anbox-support/ubuntu xenial InRelease                                  
Stary:11 http://ppa.launchpad.net/nilarimogard/webupd8/ubuntu xenial InRelease                                   
Stary:13 http://ppa.launchpad.net/nixnote/nixnote2-daily/ubuntu xenial InRelease    
Stary:14 http://ppa.launchpad.net/ozmartian/apps/ubuntu xenial InRelease
Stary:15 http://ppa.launchpad.net/pipelight/stable/ubuntu xenial InRelease
Stary:16 http://ppa.launchpad.net/pulb/basenji-daily/ubuntu xenial InRelease
Stary:17 http://ppa.launchpad.net/ravefinity-project/ppa/ubuntu xenial InRelease
Stary:18 http://ppa.launchpad.net/webupd8team/java/ubuntu xenial InRelease
Pobieranie:20 http://security.ubuntu.com/ubuntu xenial-security/universe amd64 DEP-11 Metadata [32,2 kB]
Pobieranie:21 http://security.ubuntu.com/ubuntu xenial-security/universe DEP-11 128x128 Icons [98,5 kB]
Pobieranie:22 http://security.ubuntu.com/ubuntu xenial-security/main amd64 DEP-11 Metadata [54,5 kB]
Pobieranie:23 http://security.ubuntu.com/ubuntu xenial-security/main DEP-11 128x128 Icons [120 kB] 
Pobieranie:24 http://archive.ubuntu.com/ubuntu xenial-updates/universe amd64 DEP-11 Metadata [160 kB]
Stary:25 https://deb.opera.com/opera-stable stable InRelease          
Pobieranie:12 https://origin.archive.neon.kde.org/dev/stable xenial InRelease [15,1 kB]
Pobieranie:26 http://archive.ubuntu.com/ubuntu xenial-updates/universe DEP-11 128x128 Icons [467 kB] 
Pobieranie:27 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 DEP-11 Metadata [289 kB]  
Pobieranie:28 http://archive.ubuntu.com/ubuntu xenial-updates/main DEP-11 128x128 Icons [457 kB]  
Pobieranie:29 http://archive.ubuntu.com/ubuntu xenial-updates/multiverse amd64 DEP-11 Metadata [2520 B]
Pobieranie:30 https://origin.archive.neon.kde.org/dev/stable xenial/main Sources [226 kB]           
Pobieranie:31 https://origin.archive.neon.kde.org/dev/stable xenial/main amd64 Packages [706 kB]
Pobieranie:32 https://origin.archive.neon.kde.org/dev/stable xenial/main i386 Packages [104 kB]
Pobieranie:33 https://origin.archive.neon.kde.org/dev/stable xenial/main all Packages [104 kB]
Pobrano 3038 kB w 4s (698 kB/s)                        
Czytanie list pakietów... Gotowe
Budowanie drzewa zależności       
Odczyt informacji o stanie... Gotowe
389 packages can be upgraded. Run 'apt list --upgradable' to see them.
+ sudo apt install -y anbox-modules-dkms anbox-common
Czytanie list pakietów... Gotowe
Budowanie drzewa zależności       
Odczyt informacji o stanie... Gotowe
Następujące pakiety zostały zainstalowane automatycznie i nie są już więcej wymagane:
  linux-headers-4.4.0-71 linux-headers-4.4.0-71-generic linux-headers-4.4.0-72 linux-headers-4.4.0-72-generic
  linux-image-4.4.0-71-generic linux-image-4.4.0-72-generic linux-image-extra-4.4.0-71-generic
  linux-image-extra-4.4.0-72-generic linux-tools-4.4.0-71 linux-tools-4.4.0-71-generic snap-confine
Aby je usunąć należy użyć "sudo apt autoremove".
The following additional packages will be installed:
  dbus-user-session
Zostaną zainstalowane następujące NOWE pakiety:
  anbox-common dbus-user-session
Następujące pakiety zostaną zaktualizowane:
  anbox-modules-dkms
1 aktualizowanych, 2 nowo instalowanych, 0 usuwanych i 388 nieaktualizowanych.
Konieczne pobranie 47,6 kB archiwów.
Po tej operacji zostanie dodatkowo użyte 4096 B miejsca na dysku.
Pobieranie:1 http://ppa.launchpad.net/morphis/anbox-support/ubuntu xenial/main amd64 anbox-common all 6~xenial1 [4350 B]
Pobieranie:2 http://archive.ubuntu.com/ubuntu xenial-updates/universe amd64 dbus-user-session all 1.10.6-1ubuntu3.3 [9294 B]
Pobieranie:3 http://ppa.launchpad.net/morphis/anbox-support/ubuntu xenial/main amd64 anbox-modules-dkms all 6~xenial1 [34,0 kB]
Pobrano 47,6 kB w 0s (174 kB/s)              
Wybieranie wcześniej niewybranego pakietu dbus-user-session.
(Odczytywanie bazy danych ... 664410 plików i katalogów obecnie zainstalowanych.)
Przygotowywanie do rozpakowania pakietu .../dbus-user-session_1.10.6-1ubuntu3.3_all.deb ...
Rozpakowywanie pakietu dbus-user-session (1.10.6-1ubuntu3.3) ...
Wybieranie wcześniej niewybranego pakietu anbox-common.
Przygotowywanie do rozpakowania pakietu .../anbox-common_6~xenial1_all.deb ...
Rozpakowywanie pakietu anbox-common (6~xenial1) ...
Przygotowywanie do rozpakowania pakietu .../anbox-modules-dkms_6~xenial1_all.deb ...
Rozpakowywanie pakietu anbox-modules-dkms (6~xenial1) nad (4~xenial1) ...
Konfigurowanie pakietu dbus-user-session (1.10.6-1ubuntu3.3) ...
Konfigurowanie pakietu anbox-common (6~xenial1) ...
Konfigurowanie pakietu anbox-modules-dkms (6~xenial1) ...
Loading new anbox-modules-ashmem-6~xenial1 DKMS files...
First Installation: checking all kernels...
Building for 4.4.0-75-generic and 4.4.0-77-generic
Building for architecture amd64
Building initial module for 4.4.0-75-generic
Done.

ashmem_linux:
Running module version sanity check.

Good news! Module version  for ashmem_linux.ko
exactly matches what is already found in kernel 4.4.0-75-generic.
DKMS will not replace this module.
You may override by specifying --force.

depmod............

DKMS: install completed.
Building initial module for 4.4.0-77-generic
Done.

ashmem_linux:
Running module version sanity check.

Good news! Module version  for ashmem_linux.ko
exactly matches what is already found in kernel 4.4.0-77-generic.
DKMS will not replace this module.
You may override by specifying --force.

depmod...........

DKMS: install completed.
Loading new anbox-modules-binder-6~xenial1 DKMS files...
First Installation: checking all kernels...
Building for 4.4.0-75-generic and 4.4.0-77-generic
Building for architecture amd64
Building initial module for 4.4.0-75-generic
Done.

binder_linux:
Running module version sanity check.

Good news! Module version  for binder_linux.ko
exactly matches what is already found in kernel 4.4.0-75-generic.
DKMS will not replace this module.
You may override by specifying --force.

depmod...........

DKMS: install completed.
Building initial module for 4.4.0-77-generic
Done.

binder_linux:
Running module version sanity check.

Good news! Module version  for binder_linux.ko
exactly matches what is already found in kernel 4.4.0-77-generic.
DKMS will not replace this module.
You may override by specifying --force.

depmod...........

DKMS: install completed.
+ snap info anbox
+ grep -q installed:
+ sudo snap refresh --edge anbox
error: cannot refresh "anbox": cannot get refresh information for snap "anbox": Post https://search.apps.ubuntu.com/api/v1/snaps/metadata: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
+ true
+ set +x

Done!

To ensure all changes made to your system you should now reboot
your system. If you don't do this no Android applications will
show up in the system application launcher.

anbox system-info
/bin/bash: warning: setlocale: LC_ALL: cannot change locale (pl_PL.UTF-8)
Naruszenie ochrony pamięci (zrzut pamięci)
breach of memory security

anbox version
/bin/bash: warning: setlocale: LC_ALL: cannot change locale (pl_PL.UTF-8)
anbox 0.1.0

anbox session manager just hangs 

anbox container-manager
/bin/bash: warning: setlocale: LC_ALL: cannot change locale (pl_PL.UTF-8)
[EE 2017-05-06 12:00:00] [container_manager.cpp:90@setup_mounts] Android image does not exist at path "/var/lib/anbox/android.img"

```

**Comment by morphis** [comment]
> anbox session manager just hangs 

What makes you think it hangs?

> anbox container-manager

You should never run the container-manager manually. There is a systemd service unit which takes care about this.

> [EE 2017-05-06 12:00:00] [container_manager.cpp:90@setup_mounts] Android image does not exist at path "/var/lib/anbox/android.img"

If you still want to run the container-manager you need to either place an android.img at /var/lib/anbox/android.img or provide a different path with the --android-image command line parameter. See `anbox container-manager --help` for more details.

**Comment by morphis** [comment]
> /bin/bash: warning: setlocale: LC_ALL: cannot change locale (pl_PL.UTF-8)

This looks like a real problem. Will look into that.

**Comment by omega3** [comment]
Sorry for the delay in responding. It was on KDE Neon developer edition.

**Comment by jmfGithub** [comment]
Hi,
I encounter the same problem:
anbox.appmgr
/bin/bash: warning: setlocale: LC_ALL: cannot change locale (fr_FR.UTF-8)
/bin/bash: warning: setlocale: LC_ALL: cannot change locale (fr_FR.UTF-8)

Anbox Application Manager is running but not so easy to type with my azerty (french) keyboard.

The installation was done on a Ubuntu 18.04.1 LTS
by
  sudo add-apt-repository ppa:morphis/anbox-support
  sudo aptitude update
  sudo aptitude install linux-headers-generic anbox-modules-dkms
  sudo modprobe ashmem_linux
  sudo modprobe binder_linux
  ls -1 /dev/{ashmem,binder}
  /dev/ashmem
  /dev/binder
  snap install --devmode --beta anbox

All the best

**Comment by baptx** [comment]
I got this issue because I have set the `LC_ALL` locale in `/etc/default/locale` which was required to make PostgreSQL work for another project: https://stackoverflow.com/questions/31645550/postgresql-why-psql-cant-connect-to-server/55358205#55358205.
I did not notice any issue with this warning on Anbox, it seems to work normally but it would be good to know why it happens and why it is needed.
A workaround to get rid of the warning is to set an empty value for example by executing `LC_ALL=` in the shell or service before starting `anbox session-manager` and `anbox launch --package=org.anbox.appmgr --component=org.anbox.appmgr.AppViewActivity`.

@jmfGithub Even with the warning (when using `LC_ALL=en_US.UTF-8` in  `/etc/default/locale`), my French keyboard layout continued to work normally, you can see this comment to get the correct layout: https://github.com/anbox/anbox/issues/253#issuecomment-449777527.

@morphis If I understand correctly, we should never run `anbox container-manager` manually for the official Anbox snap package since it is already done inside but if we use a fork without snap package, like for camera support, it is fine to do it manually: https://github.com/anbox/anbox/issues/727#issuecomment-877768078.
